mailer.rst

```diff
@@ -279,6 +279,30 @@ images inside the HTML contents::
         ->html('<img src="cid:logo"> ... <img src="cid:footer-signature"> ...')
     ;
 
+Catching Transport Exceptions
+-----------------------------
+
+If handing over the email to your transport fails, a
+:class:`Symfony\\Component\\Mailer\Exception\\TransportException` is thrown. To
```

Comment:
```suggestion
:class:`Symfony\\Component\\Mailer\\Exception\\TransportExceptionInterface` is thrown. To
```

This fixes the build and I think it's best to use the same class as Fabien's recommendation (and the code).